pmd-java/src/main/java/net/sourceforge/pmd/lang/java/symbols/internal/impl/ast/AbstractAstBackedSymbol.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * BSD-style license; for more info see http://pmd.sourceforge.net/license.html
+ */
+
+package net.sourceforge.pmd.lang.java.symbols.internal.impl.ast;
```

Comment:
I'm wondering, whether we really want to have both `internal` and `impl` - what's the difference? Looks to me the same...